iam/policy-templates/dss-operations-lambda.json

```diff
@@ -0,0 +1,41 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:CreateLogGroup",
+        "logs:CreateLogStream",
+        "logs:PutLogEvents"
+      ],
+      "Resource": "arn:aws:logs:*:*:*"
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
```

Comment:
We should consider not giving the lambda deletion abilities.

Reply:
Deletions on the main S3 buckets are denied externally to the data-store repo:
https://github.com/HumanCellAtlas/dcp-infra/blob/master/terraform/accounts/humancellatlas/policy_aws_replica.tf